plugins/woocommerce/src/Internal/ProductAttributesLookup/DataRegenerator.php

```diff
@@ -165,6 +167,41 @@ private function initialize_table_and_data() {
 		return true;
 	}
 
+	/**
+	 * Create the indices for the lookup table unless they exist already.
+	 */
+	public function maybe_create_table_indices() {
+		global $wpdb;
+
+		// phpcs:disable WordPress.DB.PreparedSQL.NotPrepared
+		$wpdb->query(
+			"
+IF (SELECT COUNT(1)
+	FROM INFORMATION_SCHEMA.STATISTICS
+	WHERE table_schema=' " . $wpdb->dbname . "'
+	AND table_name='" . $this->lookup_table_name . "'
+	AND index_name='product_or_parent_id_term_id')=0
+THEN
+	ALTER TABLE wp_wc_product_attributes_lookup
+	ADD INDEX product_or_parent_id_term_id (product_or_parent_id, term_id);
+END IF;"
+		);
+
+		$wpdb->query(
+			"
+IF (SELECT COUNT(1)
+	FROM INFORMATION_SCHEMA.STATISTICS
+	WHERE table_schema=' " . $wpdb->dbname . "'
+	AND table_name='" . $this->lookup_table_name . "'
+	AND index_name='is_variation_attribute_term_id')=0
+THEN
+	ALTER TABLE wp_wc_product_attributes_lookup
```

Comment:
```suggestion
	ALTER TABLE {$this->lookup_table_name}
```

Reply:
(Otherwise will fail if the prefix is not `wp_`.)

Reply:
Argh. What a rookie mistake. 😓 Fixed, thanks!